crates/runtime/src/model/chat.rs

```diff
@@ -191,7 +191,9 @@ fn azure(
 ) -> Result<Box<dyn Chat>, LlmError> {
     let Some(model_name) = model_id else {
         return Err(LlmError::FailedToLoadModel {
-            source: format!("For Azure model '{model_name}', model id must be specified in `from:azure:<model_id>`.").into(),
+            source: format!(
+    "Azure model '{model_name}' requires a model ID in the format `from:azure:<model_id>`. See https://docs.spiceai.org/components/models/azure for details."
```

Comment:
```suggestion
    "Azure model '{model_name}' requires a model ID in the format `from:azure:<model_id>`.\nFor details, visit: https://docs.spiceai.org/components/models/azure"
```

Links with additional context [should be on newlines](https://github.com/spiceai/spiceai/blob/trunk/docs/dev/error_handling.md). Let's also align with the existing wording where links appear like `For details, visit: ...`